tests/conftest.py

```diff
@@ -29,41 +29,49 @@ def node(host, request):
         'mimic': 13,
         'dev': 99
     }
-    if not request.node.get_marker(node_type) and not request.node.get_marker('all'):  # noqa E501
-        pytest.skip("Not a valid test for node type: %s" % node_type)
 
-    if request.node.get_marker("no_lvm_scenario") and lvm_scenario:
+    # capture the initial/default state
+    test_is_applicable = False
+    for marker in request.node.iter_markers():
+        if marker.name in group_names or marker.name == 'all':
+            test_is_applicable = True
```

Comment:
I think we must break this loop as soon as we enter in this condition, otherwise, we can reset it to `False`.
I guess this is kind of what @alfredodeza said [here](https://github.com/ceph/ceph-ansible/pull/2792#discussion_r197156568)

Reply:
Breaking out of the loop isn't really necessary, as this assignment is not going to change it back to `False`. If you really want to break out of it though, that is fine. It wouldn't be any different